widgetssdk/src/main/java/com/glia/widgets/base/FadeTransitionActivity.kt

```diff
@@ -34,4 +35,13 @@ open class FadeTransitionActivity : AppCompatActivity() {
         overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
     }
 
+    fun setTitle(locale: LocaleString?) {
+        if (locale == null) {
+            super.setTitle(null)
+            return
+        }
+
+        // TODO: handle locale updated
```

Comment:
Is it a leftover, or it's really out of the current scope?

Reply:
Actually forgot to implement

Reply:
Now should be fixed